engine/events/overworld.asm

```diff
@@ -0,0 +1,100 @@
+INCLUDE "constants.asm"
+
+SECTION "TeleportFunction", ROMX[$52db], BANK[$03]
+
+TeleportFunction: ; 03:52db
+	xor a
+	ld [wFieldMoveScriptID], a
+.loop
+	ld a, [wFieldMoveScriptID]
+	bit 7, a

```

Comment:
This bit means something. What does it mean? Make a constant. Constantize all of its uses by or-ing it with the script ID value.